book/website/reproducible-research/rdm/rdm-visualisation.md

```diff
@@ -11,26 +11,36 @@ Therefore, data visualisation is an aspect of research data management!
 
 Below follow some resources that may help you in creating clearer, more accessible and more transparant data visualisations. 
 
+<!--
+```{figure} ../figures/data-visualisation.*
+---
+name: project-design
+alt: A group of people collaboratively developing a data visualisation by using a giant pencil. 
+---
+ _The Turing Way_ project illustration by Scriberia. Used under a CC-BY 4.0 licence. DOI: [10.5281/zenodo.3332807](https://doi.org/10.5281/zenodo.3332807).
+```
+-->
+
 (rr-rdm-visualisation-tools)=
 ## Tools
 
-- [upset graphs](https://upset.app/)
-- Python plotly
-  - [annotating visualisations in python plotly](https://medium.com/nerd-for-tech/enriching-data-visualizations-with-annotations-in-plotly-using-python-6127ff6e0f80) (blog and video)
+-  [upset graphs](https://upset.app/) are a straightforward way to visualize set intersections in a matrix layout, which can help in analysing multiple datasets at once.
+-  In Python Plotly, annotations and animations can serve as extra visual cues to highlight important features.
```

Comment:
Is that really specific to python plotly? Should definetely work also in javascript. Not sure what the state of the other backends is

Reply:
Fair point - I rewrote it to reflect that this is pointing things out for Plotly and not speaking in general!